test/Infer/demanded-bits1.opt

```diff
@@ -0,0 +1,9 @@
+; REQUIRES: solver, solver-model
+
+; RUN: %souper-check %solver -infer-rhs -souper-infer-iN %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: Failed to infer RHS
```

Comment:
wait, Souper is supposed to successfully synthesize a constant here. it can be any constant so the job is easy. none of the bits are demanded so any value works.